lib/clis/wolkenkit/dev/devCommand.ts

```diff
@@ -126,6 +126,8 @@ const devCommand = function (): Command<DevOptions> {
           env: {
             ...processenv() as NodeJS.ProcessEnv,
             APPLICATION_DIRECTORY: applicationDirectory,
+            HTTP_API: String(true),
```

Comment:
Here I'd also suggest to prefix these variables with something such as `ENABLE_` to better show off that this is a boolean value:

```suggestion
            ENABLE_HTTP_API: String(true),
```

Reply:
They aren't all boolean values though. The graphql configuration is either false or a configuration object. This might be the case for the http API in the future as well. I'd leave it as-is, since people deploying the wolkenkit will have to take a look at the environment variable documentation anyway.